synology_dsm/synology_dsm.py

```diff
@@ -366,6 +382,9 @@ def reset(self, api):
         if isinstance(api, SynoStorage):
             self._storage = None
             return True
+        if isinstance(api, SynoCoreSystem):
+            self._system = None
+            return True

```

Comment:
Also order + add the same for `SynoCoreUpgrade` instance

Reply:
done